src/utils/event-utils.ts

```diff
@@ -32,6 +32,16 @@ export const CAMPAIGN_PARAMS = [
 ]
 
 export const Info = {
+    location: function (): { href?: string; hrefIncludingHash?: string; host?: string; pathname?: string } {
+        return {
+            // NOTE: Hash params are typically sensitive and not designed to be sent to servers so we exclude them here by default
+            href: location?.href?.split('#')[0],
+            hrefIncludingHash: location?.href,
```

Comment:
almost feels like we should capture the hash params separately so that if folk really want them they can capture them

instead of having href _and_ hash

I don't think that massively affects queries so no strong opinion but would sit nicely to have `query_params`, `pathname`, and `hash_params` as separate things

(no need to take that as blocking here i think generally we could do with better url handling)